src/cmd/build.js

```diff
@@ -118,6 +120,12 @@ export class FileFilter {
         return false;
       }
     }
+    for (const filePath of this.filePathsToIgnore) {
+      if (filePath === path){
+        log.debug(`FileFilter: ignoring file ${path}`);
```

Comment:
I'd like to see "ignoring file" change to "ignoring file path" so that we can differentiate it from the other debug message.